rs/rosetta-api/icp/client/src/lib.rs

```diff
@@ -818,6 +848,33 @@ impl RosettaClient {
         )
         .await
     }
+
+    /// If a neuron is in the state DISSOLVING you can disburse the neuron with this function.
```

Comment:
Should this be state DISSOLVED?

Reply:
Technically DISSOLVED does not exist as a state. There only exists DISSOLVING with a DissolveState that indicates that the timestamp at which the neuron dissolves is in the past.

Reply:
I see, thanks for the clarification!